File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/Category.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.arch.persistence.room.Entity;
import android.os.Parcel;
import android.os.Parcelable;

import java.util.List;
@Entity(primaryKeys = {"id"}, tableName = "spotify_category")
public class Category {
  public String href;
  public List<Image> icons;
  public String id;
  public String name;

  public Category() {
  }

}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/Category.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.arch.persistence.room.Entity
import android.os.Parcel
import android.os.Parcelable

@Entity(primaryKeys = arrayOf("id"), tableName = "spotify_category")
class Category {
    var href: String = ""
    var icons: List<Image> = listOf()
    var id: String = ""
    var name: String = ""

}
